lua/wikis/commons/Widget/Patch/List.lua

```diff
@@ -200,7 +200,7 @@ end
 ---@return Widget?
 function PatchList:_monthHeaderRow(patch)
 	local month = DateExt.formatTimestamp('M', patch.releaseDate.timestamp)
-	local year = DateExt.formatTimestamp('Y', patch.releaseDate.timestamp)
+	local year = patch.releaseDate.year
```

Comment:
Wouldn't it be better for month too? Or is month using the month name instead of num?

Reply:
'M' doesn't return the month number but the month abbreviation, e.g. `Apr`